uk.co.compucorp.civicrm.hremails/email-templates/src/partials/request-data.html

```diff
@@ -0,0 +1,10 @@
+<row class="{{class}}">
+  <columns class="request-data-key" large="{{#if keySize}}{{keySize}}{{else}}3{{/if}}">
```

Comment:
Unfortunately no. There is no ternary operator in Handlebars, and even adding some whitespaces
```html
{{#if keySize}} {{keySize}} {{else}} 3 {{/if}}
```
results in the `if` not working anymore